react/week3/timeregistration/src/TimeCalculator.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
```

Comment:
same here whats the prop-types and whats the use of it ?

Reply:
The homework stated that we must demostrate that we know how to use prototypes on components eg proptypes.shape, proptypes.strings, proptypes.number,.,.see "Technical details
All components should have prop types defined."